base/multimedia.jl

```diff
@@ -57,31 +57,33 @@ mimewritable(m::AbstractString, x) = mimewritable(MIME(m), x)
 # format and is returned unmodified.  This is useful so that raw data can be
 # passed to display(m::MIME, x).
 
+verbose_writemime(io, m, x) = writemime(IOContext(io,multiline=true,limit=false), m, x)
```

Comment:
Note that this led to JuliaLang/IJulia.jl#462

It's not a big deal; in hindsight, I'm not sure the whole `stringmime` function belongs in Base anyway.  It should be easy enough to implement a `limitstringmime` in IJulia.